SMLHelper/Handlers/CraftDataHandler.cs

```diff
@@ -21,6 +22,69 @@ private CraftDataHandler()
 
         #region Static Methods
 
+#if BELOWZERO
```

Comment:
Split `CraftDataHandler.cs` into multiple files for SN and BZ, this is getting messy.

Reply:
This will be handled by @MrPurple6411 and @ahk1221